app/src/main/java/pl/droidcon/app/reminder/ReminderImpl.java

```diff
@@ -46,6 +46,6 @@ public void removeRemind(@NonNull Session session) {
 
     private PendingIntent createIntentForRemind(Session session) {
         Intent intent = ReminderReceiver.createReceiverIntent(context, session);
```

Comment:
Popatrzylem na implementacje `ReminderReceiver. createReceiverIntent` i wrzucamy do `PendingIntent` customowy obiekt `parcelable`,  od Androida 7 nie bedzie to dzialalo, tak jak jest opisane na stacku http://stackoverflow.com/questions/39573849/android-7-intent-extras-missing.

Zrobie dzis wieczorem PR na to.